nightfall-deployer/src/contract-setup.mjs

```diff
@@ -4,96 +4,62 @@ address of the contract that holds global state (State.sol)
 */
 
 import config from 'config';
-import logger from 'common-files/utils/logger.mjs';
 import Web3 from 'common-files/utils/web3.mjs';
 import { waitForContract } from 'common-files/utils/contract.mjs';
+import logger from 'common-files/utils/logger.mjs';
 
 async function setupContracts() {
+  const web3 = Web3.connection();
+
   const stateInstance = await waitForContract('State');
   logger.debug(`address of State contract is ${stateInstance.options.address}`);
 
-  const simpleMultiSigAddress = (await waitForContract('SimpleMultiSig')).options.address;
-  const shieldContractInstance = await waitForContract('Shield');
-  const stateContractInstance = await waitForContract('State');
-  const proposerContractInstance = await waitForContract('Proposers');
-  const challengesContractInstance = await waitForContract('Challenges');
+  const tx = {
+    from: process.env.FROM_ADDRESS,
+    // value: fee,
+    gas: config.WEB3_OPTIONS.gas,
+    gasPrice: config.WEB3_OPTIONS.gasPrice,
+  };
 
-  logger.debug(
-    `transfering ownership of contracts to simpleMultiSigAddress ${simpleMultiSigAddress}`,
-  );
+  const proposers = await waitForContract('Proposers');
```

Comment:
I would put the names of the variables with `Instance` as the first one above `stateInstance` to clarify they are instance of the contract.
So `proposersInstance`, `shieldInstance`, `challengesInstance`.